lms/djangoapps/gating/api.py

```diff
@@ -61,7 +61,7 @@ def _get_subsection_percentage(subsection_grade):
     """
     Returns the percentage value of the given subsection_grade.
     """
-    return _calculate_ratio(subsection_grade.graded_total.earned, subsection_grade.graded_total.possible) * 100.0
+    return int(round(_calculate_ratio(subsection_grade.graded_total.earned, subsection_grade.graded_total.possible) * 100.0))
```

Comment:
Are we keeping in mind that we need to make sure that rounding is consistent on all 3 views?
Secondly, this PR needs tests.

Reply:
Changed to ensure consistent rounding across all 3 views.